migrate_nidirect_utils/src/Command/NidirectMigratePreCommand.php

```diff
@@ -199,4 +199,40 @@ protected function task_remove_recipe_nodes() {
     $this->drupal7DatabaseQuery("DELETE FROM node WHERE node.type = 'nidirect_recipe'");
   }
 
+  /**
+   * Remove umbrella body node and taxonomy path aliases.
+   */
+  // phpcs:disable
+  protected function task_remove_umbrella_body_url_aliases() {
+  // phpcs:enable
+    $this->drupal7DatabaseQuery("DELETE FROM url_alias WHERE url_alias.alias LIKE 'accessni/%' OR url_alias.alias LIKE 'accessni-%'");
+  }
+
+  /**
+   * Remove umbrella body node metatags.
+   */
+  // phpcs:disable
+  protected function task_remove_umbrella_body_node_metatags() {
+  // phpcs:enable
+    $this->drupal7DatabaseQuery("DELETE FROM metatag INNER JOIN node ON node.nid = metatag.entity_id WHERE node.type = 'nidirect_ub'");
+  }
+
+  /**
+   * Remove umbrella body node revisions.
+   */
+  // phpcs:disable
```

Comment:
+1 for selective use of this. phpcs can be a bit fussy with Drupal console commands